layouts/partials/footer.html

```diff
@@ -12,28 +12,32 @@
       <div class="col-6 col-sm-4 text-right text-xs-center order-sm-3">
         {{ with $links }}
         {{ with index . "developer"}}
-        {{ template "footer-links-block"  . }}
-        {{ end }}
+        <ul class="list-inline mb-0">
+          {{ template "footer-links-block"  . }}
+            <li class="list-inline-item mx-2" >&nbsp</li>
+            <li class="list-inline-item mx-2" >
+              <img src="/img/powered-gvisor.png" alt="Powered by gVisor">
+            </li>
+          {{ end }}
+        </ul>
         {{ end }}
       </div>
       <div class="col-12 col-sm-4 text-center py-2 order-sm-2">
         {{ with .Site.Params.copyright }}<small class="text-white">&copy; {{ now.Year}} {{ .}} {{ T "footer_all_rights_reserved" }}</small>{{ end }}
         {{ with .Site.Params.privacy_policy }}<small class="ml-1"><a href="{{ . }}">{{ T "footer_privacy_policy" }}</a></small>{{ end }}
-	{{ if not .Site.Params.ui.footer_about_disable }}
-		{{ with .Site.GetPage "about" }}<p class="mt-2"><a href="{{ .RelPermalink }}">{{ .Title }}</a></p>{{ end }}
-	{{ end }}
+          {{ if not .Site.Params.ui.footer_about_disable }}
```

Comment:
I don't think this line's indentation is correct. It should match the line above and its associated end.